frontend/src/metabase-lib/v1/expressions/formatter/formatter.ts

```diff
@@ -1,6 +1,6 @@
 import type { AstPath, Doc, ParserOptions, Plugin } from "prettier";
 import { builders } from "prettier/doc";
-import { format as pformat } from "prettier/standalone";
+import { format as staticPrettierFormat } from "prettier/standalone";
```

Comment:
When the `process.env.IS_EMBEDDING_SDK` is defined, this import is removed by a bundler